setup/gcp/install_setup.py

```diff
@@ -34,12 +35,15 @@ def run():
     parser.add_argument('--advanced',
                         action='store_true',
                         help='Advanced setup mode (more options)')
-
     parser.add_argument('--dry-run',
                         action='store_true',
                         help=('Generate config files but do not modify '
                               'GCP infrastructure (i.e. do not actually '
                               'set up Forseti)'))
+    parser.add_argument('--type',
+                        choices=['client', 'server'],
+                        help='Type of the installation, '
```

Comment:
In the help, can you please add, "If type is not specified, then both server and client will be deployed."